test/helpers/cons.go

```diff
@@ -168,7 +168,7 @@ const (
 	DefaultNamespace       = "default"
 	KubeSystemNamespace    = "kube-system"
 	CiliumNamespaceDefault = KubeSystemNamespace
-	CiliumNamespaceGKE     = "cilium"
+	CiliumNamespaceGKE     = KubeSystemNamespace
```

Comment:
This variable can be removed and `GetCiliumNamespace()` can be simplified or maybe even removed.